src/PrefsAct.scala

```diff
@@ -33,7 +37,10 @@ class PrefsAct extends PreferenceActivity {
 
 			UIHelper.shareFile(this, file, filename)
 		} catch {
-			case e : Exception => Toast.makeText(this, e.getMessage(), Toast.LENGTH_LONG).show()
+			case e : Exception => {
+				Toast.makeText(this, e.getMessage(), Toast.LENGTH_LONG).show()
+				Log.i(TAG, "Caught exception sharing file: " + e, e)
```

Comment:
Not sure if this shouldn't be `Log.e` or at least `Log.w` ;)

Reply:
Good point! Probably any errors that warrant generating an on-screen toast message should probably be warn or higher.

Reply:
After further thought, if we ever hit that catch, it basically means that a piece of user functionality was not able to do its job so it should probably be an error-level log. I can push an update this evening if preferred or, I think this could even be edited from the web though without the GitHub CI branch in place to automatically compile and test, I don't know if I'd trust that.